src/length.js

```diff
@@ -0,0 +1,45 @@
+import {deg2rad} from "./math";
+import {default as geoStream} from "./stream";
+
+var lengthSum;
+
+var noop = function() {};
```

Comment:
And actually, I’d probably put noop in its own file, since such a thing tends to be used in multiple places.